docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
                 - INSTALL_WORKSPACE_SSH=false
                 - PUID=1000
                 - PGID=1000
-                - NODE_VERSION=stable
+                - NODE_VERSION=6
```

Comment:
Not always new major version fully backward compatible (see on modes of mysql 5.7)
It not very good if with enabling xdebug I bring new version of node and something crash...